crates/bevy_ecs/src/schedule/mod.rs

```diff
@@ -206,7 +210,12 @@ impl Schedule {
             let _stage_guard = stage_span.enter();
             let stage = self.stages.get_mut(label).unwrap();
             stage.run(world);
+            world
+                .scheduler_commands
+                .transfer(&mut commands, existing_commands);
```

Comment:
This seems to be the only place `.transfer()` is used. Maybe we don't need to do this at all, and instead apply commands directly from the world, at the end of this function's scope? Won't need to allocate a new queue (or hold onto one in the schedule) either.

Reply:
Good catch! I missed it while changing around everything else.